poetry.lock

```diff
@@ -1,25 +1,27 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
@henriquebastos 2 things:

1. We are still using 1.8
2. Are there any real changes in here? If so, can this be it's own PR?

Reply:
Reverted. I was having trouble setting up my environment and it was fixed by updating this.